jetty-core/jetty-http3/jetty-http3-server/src/main/java/org/eclipse/jetty/http3/server/internal/HTTP3SessionServer.java

```diff
@@ -117,4 +118,18 @@ private void notifyAccept()
             LOG.info("failure notifying listener {}", listener, x);
         }
     }
+
+    Stream.Server.Listener notifyRequest(HeadersFrame frame)
+    {
+        Server.Listener listener = getListener();
```

Comment:
Can/should the listener be null? Shouldn't there be a null check in the constructor, or in the super constructor?